docs/data/data-grid/components/toolbar/GridToolbar.js

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { DemoContainer } from '@mui/x-data-grid/internals/demo';
+import { Grid, GridToolbarQuickFilter } from '@mui/x-data-grid';
+import ViewColumnIcon from '@mui/icons-material/ViewColumn';
+import FilterListIcon from '@mui/icons-material/FilterList';
+import PrintIcon from '@mui/icons-material/Print';
+import FileDownloadIcon from '@mui/icons-material/FileDownload';
+import GridViewIcon from '@mui/icons-material/ViewModule';
+import ListViewIcon from '@mui/icons-material/ViewList';
+
+export default function GridToolbar() {
+  const [view, setView] = React.useState('grid');
+  return (
+    <DemoContainer>
+      <Grid.Toolbar.Root sx={demoStyles}>
+        <Grid.Toolbar.Button aria-label="Columns">
+          <ViewColumnIcon fontSize="small" />
+        </Grid.Toolbar.Button>
+
+        <Grid.Toolbar.Button aria-label="Filters">
+          <FilterListIcon fontSize="small" />
+        </Grid.Toolbar.Button>
+
+        <Grid.Toolbar.Separator />
+        <Grid.Toolbar.Button aria-label="Print">
+          <PrintIcon fontSize="small" />
+        </Grid.Toolbar.Button>
+
+        <Grid.Toolbar.Button aria-label="Download">
+          <FileDownloadIcon fontSize="small" />
+        </Grid.Toolbar.Button>
+
+        <GridToolbarQuickFilter sx={{ ml: 'auto', mr: 0.5 }} />
+        <Grid.Toolbar.ToggleButtonGroup value={view}>
+          <Grid.Toolbar.ToggleButton
```

Comment:
There seems to be no added value in these components, just an unnecessary wrapper.
I'm not sure we need them 🤔
WDYT?